Spreadsheets (#1783)
enkidevs/curriculum: excel/beginner/basics-of-excel/what-is-a-sheet.md

---
author: Stefan-Stojanovic

levels:
  - beginner

aspects:
  - new
  - workout

type: normal

category: how to

---

# What is Sheet?

---
## Content

> A Sheet or a Worksheet in Excel is a single page(sheet) of a Workbook/Spreadsheet/file.

Reviewer: ```suggestion
A Sheet or a Worksheet in Excel is a single page (sheet) of a Workbook/Spreadsheet/file.
```

It consists of rows usually marked with numbers and columns usually marked with letters.

The maximum number of rows, columns, and cells a sheet can have, solely depends on the version of Excel.

> For instance, a Google docs sheet can have a maximum of 500,000 rows, 18,278 columns or 5,000,000 cells. As soon as the cell count reaches 5milion, you cannot make any more new rows or columns.

Reviewer: ```suggestion
For instance, a Google sheet can have a maximum of 500,000 rows, 18,278 columns or 5,000,000 cells. As soon as the cell count reaches 5 million, you cannot make any more new rows or columns.
```

To add a new sheet in Google docs, go to the bottom left corner of the spreadsheet and press on the `+` sign.
The process is the same for Microsoft Excel.

![image.png](https://img.enkipro.com/0ea4f3f810b4934f42ddb4d704184a78.png)

> Right-clicking a sheet opens a menu where you can rename it, move it, duplicate, delete and more.

Reviewer: ```suggestion
Right-clicking a sheet opens a menu where you can rename it, move it, duplicate it, delete it and more.
```
